pkg/controllers/csi/driver/server.go

```diff
@@ -217,6 +217,26 @@ func (svr *Server) unmount(volumeInfo csivolumes.VolumeInfo) {
 
 		_ = svr.fs.RemoveAll(appMountDir) // try to cleanup fully, but lets not spam the logs with errors
 	}
+
+	podInfoSymlinkPath := svr.findPodInfoSymlink(volumeInfo) // cleaning up the pod-info symlink here is far more efficient instead of having to walk the whole fs during cleanup
+	if podInfoSymlinkPath != "" {
+		_ = svr.fs.Remove(podInfoSymlinkPath)
+	}
+}
+
+func (svr *Server) findPodInfoSymlink(volumeInfo csivolumes.VolumeInfo) string {
```

Comment:
I know it was before :) but it should be `srv` not `svr` :))

Reply:
done

Reply:
done